dataFields/ABFieldNumberCore.js

```diff
@@ -276,6 +276,7 @@ module.exports = class ABFieldNumberCore extends ABField {
 
    format(rowData) {
       if (
+	 typeof rowData == "undefined" ||
          rowData[this.columnName] == null ||
```

Comment:
or `rowData?.[this.columnName] == null ||` ?

Reply:
Oh...yes. this is best. Matches the same change we have made elsewhere.

Reply:
I actually need both because the ? won't help if rowData is null only if rowData.columnName is null. Right?

Reply:
Or...if rowData.columnName doesn't exist?